ReactSkia/sdk/RNSKeyCodeMapping.h

```diff
@@ -33,9 +33,9 @@ typedef enum rnsKey{
     RNS_KEY_Guia,
     //Navigation Keys
     
+    RNS_KEY_Select,
```

Comment:
Can we have Key_Type_Start & Key_TYPE_End enum to confine to pack the keys in groups .
Ex:-RNS_RCU_KEY_Start,RCU_KEY_End,RNS_NAV_KEY_STRAT,RNS_NAV_KEY_END,RNS_KB_KEY_START,RNS_KB_KEY_END
I feel, Which may give flexibility to modify the key support in future .